### DISCUSSION TITLE
Open URL icon

### DISCUSSION THREAD

**Discussion by Glubzor** [keepassxreboot/keepassxc#10064] [Ideas]
Could we have the Open URL icon in the toolbar, even in compact mode. TIA.

**Comment by droidmonkey** [comment]
In the preview pane, you can click the link. Press Ctrl+Shift+U to open a url as well. We are trying not to clutter the toolbar.

**Reply by Glubzor** [reply]
I already know these two ways of opening a link. The open URL icon could replace the copy URl icon.  (To copy URL one can right click on it... )
Thanks for not dodging with the cluttering of the toolbar.
Regards.

**Reply by droidmonkey** [reply]
I do like the idea of replacing the copy url with open url (the more common option). Especially if we introduce open in alt browser and incognito. It could have a drop down like the Auto-Type button.

**Reply by Glubzor** [reply]
Very good idea opening in alternate(s) browser(s) in such a simple way. Thanks for maintaining this excellent software.
Best regards.
